source/PlayerInfo.h

```diff
@@ -304,6 +311,7 @@ class PlayerInfo {
 	std::set<const System *> seen;
 	std::set<const System *> visitedSystems;
 	std::set<const Planet *> visitedPlanets;
+	mutable std::map<const System *, bool> seenCache, nameKnownCache;
```

Comment:
Declare one name (only) per declaration.
Multiple names per declaration is considered bad form in C++ (and may even be against ES style guide).